freqtrade/rpc/telegram.py

```diff
@@ -421,9 +422,17 @@ def _profit(self, update: Update, context: CallbackContext) -> None:
         stake_cur = self._config['stake_currency']
         fiat_disp_cur = self._config.get('fiat_display_currency', '')
 
+        start_date = datetime.fromtimestamp(0)
+        if context.args:
+            if 'day' in context.args:
+                start_date = datetime.combine(date.today(), datetime.min.time())
+            elif 'week' in context.args:
+                start_date = datetime.combine(date.today(), datetime.min.time()) - timedelta(days=7)
```

Comment:
i'm ... undecided yet if this is a good way to approach this 

i found the "week" calculation surprising (i was assuming a week that resets on a specific day ... not a rolling 7 day week) - but i guess that's an expectation problem more than anything

Combining the week calculation with day - i was somehow expecting the "day" to be a rolling 24h window ... so this seems somehow inconsistent.

Would a simpler / more flexible way be `/profit 5` (to get the profit of the last 5 days?)? 

depending on how markets behaved, having this in a dynamic way (maybe i want it for the last 2 weeks - or for the last 3 days (because 3 days ago event X happened ...)

Reply:
About week: agree, I also would like to have stats starting from the beginning of the week but in this case first day of the week should be configurable (Sunday or Monday)

/profit 5 would also work but AFAIU it's not possible to add such action to the Telegram keyboard, right?

Reply:
well - not now (not without a change ...) 

but nothing prevents you from checking via regex instead of static values (basically for `'/profit', '/profit \d+`) ...